### PR TITLE
Update README with current usage instructions

### PR DESCRIPTION
**Pull request by chrislloyd**

### CODE REVIEW

**Comment by chrislloyd** [comment]
@christianvuerings you can review it easier here: https://github.com/chrislloyd/corkboard/blob/readme/README.md

File: README.md [hunk 1]
```diff
@@ -45,182 +36,88 @@ yarn add corkboard
...
-```javascript
-group('Basics', () => {
-  ns('Colors');
-  card('Primary Colors', ...); // Goes in the "Basics" -> "Getting Started" group
-});
+group('Button');
+card(<Button color="black" />);
 ```
 
+When you use `group` like this, any subsequent cards will be place in that group and that group will appear in the navigation on the left.
```
Comment (christianvuerings) [inline README.md]: @chrislloyd perhaps leave out `on the left` since I could see themes where this wouldn't be the case.

File: README.md [hunk 2]
```diff
@@ -45,182 +36,88 @@ yarn add corkboard
...
+import { group, doc } from 'corkboard';
 
+group('Button');
 
-## Customizing Corkboard
+doc`
+# Buttons
 
-### Themes
+Buttons are useful because they lets users trigger actions in your system. They want something to happen so they click a button.
```
Comment (christianvuerings) [inline README.md]: @chrislloyd just a styling thing, would look better in GitHub when we add a breaking line after the `.`

File: README.md [hunk 3]
```diff
@@ -45,182 +36,88 @@ yarn add corkboard
...
-Array<{|
-  path: Array<string>,
-  position: Array<number>,
-  fn: () => ReactElement,
-|}>
-```
+### Titles
 
-Which often ends up looking like:
+As you can see from the previous example, it's a common pattern to create a group, then to have a card with some sort of `<h1>` to display that group. Well, Corkboard comes with a handy function to combine both of those: `title`.
```
Comment (christianvuerings) [inline README.md]: @chrislloyd might be good to monospace `group` like we do elsewhere in this `README`.

Comment (chrislloyd) [inline README.md]: I'm monospacing the function call `group` but leaving the "logical" group normal.

**Pull request merged**